.build/images/Quartz64/quartz64_defconfig

```diff
@@ -44,6 +44,7 @@ CONFIG_BLK_DEV_NBD=m
 CONFIG_BLK_DEV_NVME=y
 CONFIG_BLK_DEV_RAM=n
 CONFIG_BLK_DEV_SD=y
+CONFIG_BLK_DEV_THROTTLING=y
```

Comment:
That one seems to be pretty special, are you sure it is really used? E.g. the RPi kernel does not define it as well. Probably it requires hardware support as well.